Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'slim'
+gem 'materialize-sass'
```

Comment:
I have already added a css framework through a cdn, so this isn't really doing anything

Reply:
I have attachment issues with what I'm familiar with... I'll get over it.